app/models/ability.rb

```diff
@@ -3,17 +3,30 @@ class Ability
 
   def initialize(user)
     user ||= User.new
+    grant_anon_perms
+    grant_normal_perms(user)
+    grant_admin_perms(user) if user.id == 1
```

Comment:
`grant_admin_perms` is defined without a parameter, this causes an error when you go to create an oauth application.

Reply:
oops!